src/exchange/adl_handler.cairo

```diff
@@ -108,7 +117,14 @@ mod AdlHandler {
     //                              STORAGE
     // *************************************************************************
     #[storage]
-    struct Storage {}
+    struct Storage {
+        data_store: IDataStoreSafeDispatcher,
+        event_emitter: IEventEmitterSafeDispatcher,
+        oracle: IOracleSafeDispatcher,
+        chain: IChainSafeDispatcher,
```

Comment:
why chain?

Reply:
Removed leftover code that originated from a previous closed pull request.